typedargs/typeinfo.py

```diff
@@ -235,6 +237,26 @@ def _is_factory(cls, typeobj):
 
         return False
 
+    @staticmethod
+    def get_type_name(type_class: type) -> Union[str, None]:
```

Comment:
`Union[str, None]` is best referred to as `Optional[str]`, which is also present in the `typing` module.

Reply:
done